### PR TITLE
Removed checking by userName from getSubscription method

### PR DESCRIPTION
**Pull request by ValentinTyhonov**
Removed checking by userName from getSubscription method -> now every user can view every subscription, but they still can remove or modify only own subscriptions.

Removed checking by userName from doExistSubscription method -> it was bad idea to check it be userName, because then user could create subscription with the SubscriptionName (id), even if other one is already creted with the same name  by other user.

### CODE REVIEW

File: src/main/java/com/ericsson/ei/services/SubscriptionService.java [hunk 1]
```diff
@@ -79,20 +81,18 @@ public Subscription getSubscription(String name, String userName) throws Subscri
                 return subscription;
                 // return mapper.readValue(input, Subscription.class);
```
Comment (tobiasake) [inline src/main/java/com/ericsson/ei/services/SubscriptionService.java]: Remove this out-commented line if its not going to be used.

File: src/main/java/com/ericsson/ei/services/ISubscriptionService.java [hunk 2]
```diff
@@ -44,19 +44,21 @@
...
+    Subscription getSubscription(String name) throws SubscriptionNotFoundException;
     
     /**
      * 
      * @param subscription
      * @param subscriptionName
+     * @param user
      * @return
      */
     boolean modifySubscription(Subscription subscription, String subscriptionName, String user);
```
Comment (tobiasake) [inline src/main/java/com/ericsson/ei/services/ISubscriptionService.java]: I don't understand why we need user as input. An existing Subscription should have userName field already. And subscriptionName is the key for the unique subscription.  Why do we need user parameter? It should be removed in my point of view. Or am I missing something?

Comment (tobiasake) [reply src/main/java/com/ericsson/ei/services/ISubscriptionService.java]: Do we use 'user', username, in more places where its not needed?

Comment (ValentinTyhonov) [reply src/main/java/com/ericsson/ei/services/ISubscriptionService.java]: here we use userName for preventing modify subscription if current user is not owner of this subscription.
only in two places user param was unneeded, so there I removed this param

Comment (tobiasake) [reply src/main/java/com/ericsson/ei/services/ISubscriptionService.java]: Ok,, and the user value comes from the RestApi authentication mechanism. Then its ok.

**Pull request merged**